src/get-survey-results.js

```diff
@@ -33,7 +33,8 @@ async function waitForBuildToComplete(token, dataCenter, progressId) {
   if (data.result.percentComplete === 100) {
     return data.result.file;
   }
-
+  //wait for a second
+  Atomics.wait(new Int32Array(new SharedArrayBuffer(4)), 0, 0, 1000);
```

Comment:
I think `Atomics.wait()` will block the event loop, but I'm not sure if that's a problem in this context.